src/v/storage/tests/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ rp_test(
   BINARY_NAME storage_log_replayer_single_thread
   SOURCES
     log_replayer_test.cc
+    index_state_test.cc
```

Comment:
I don't see how this correlates with the comment, this is still a seastar thread test binary,

Reply:
you are right, sorry